lib/datadog/core/telemetry/request.rb

```diff
@@ -31,6 +31,12 @@ def build_payload(event, seq_id)
 
           def application
             config = Datadog.configuration
+
+            tracer_version = Core::Environment::Identity.gem_datadog_version_semver2
+            if defined?(::Datadog::CI::VERSION) && config.respond_to?(:ci) && config.ci.enabled
+              tracer_version = "#{tracer_version}-ci-#{::Datadog::CI::VERSION::STRING}"
+            end
```

Comment:
It may be worth adding here a bit of context to why we're doing this. Something closer to the commit description?

Since this is "suspicious", I think it's the kind of thing that deserves a bit of a note acknowledging that.